app/components/chartist-chart.js

```diff
@@ -96,6 +84,36 @@ export default Ember.Component.extend({
       this.set('type', 'line');
     }
 
-    this._super();
-  }
+    this._super(...arguments);
+  },
+
+  // This is where the business happens. This will only run if checkForReqs
+  // doesn't find any problems.
+  didInsertElement() {
+    let data = this.get('data') || this.get('defaultDataStructure');
+
+    let chart = new Chartist[this.get('chartType')](
+      this.get('element'),
+      data,
+      this.get('options'),
+      this.get('responsiveOptions')
+    );
+
+    chart.on('created', this._createdEventHook);
+
+    this.set('chart', chart);
+
+    this._super(...arguments);
+  },
+
+  onData: observer('data', function() {
```

Comment:
I'm fairly certain we can refactor away from this observer in a future pull request...